modules/standard/DateTime.chpl

```diff
@@ -471,42 +471,42 @@ module DateTime {
 
   /* Operators on date values */
   pragma "no doc"
-  proc +(d: date, t: timedelta): date {
+  operator +(d: date, t: timedelta): date {
```

Comment:
Should these be method operators  on the `date` type? They're in the "Operators on date values" section. Same question for things in the "Operators on datetime values" and "Operators on time values" sections.

Reply:
I took a conservative approach on making operators into operator methods, but I think it would make sense to make more of them into operator methods as well :)

Reply:
Yeah, that's accurate.  Basically, they can be but they fell on that boundary case for me which is why I didn't do them in the initial pass.  Since David is the one that implemented them, I'm inclined to make them into methods at his suggestion.

Reply:
Totally agree, and I would've done the same.  My questions were aimed at David to get his thoughts.  I think we could proceed with the PR without such cases being finalized as long as we keep a list of such cases that we want to handle or reconsider in a subsequent pass.